subsys/nrf_security/Kconfig.legacy

```diff
@@ -345,19 +345,9 @@ config MBEDTLS_HMAC_DRBG_C
 	  This setting will enable HMAC_DRBG APIs in mbed TLS.
 	  Corresponds to MBEDTLS_HMAC_DRBG_C setting in mbed TLS config file.
 
-config MBEDTLS_PSA_CRYPTO_EXTERNAL_RNG
-	bool
-	prompt "External RNG"
-	default y if CRYPTOCELL_USABLE
-	help
-	  Convenience layer for handling entropy before it is fully supported in
-	  PSA Crypto APIs and drivers. This is set by default for legacy and PSA
-	  modes.
-
```

Comment:
I see this has a prompt and we have a patch to openthread. Is it possible for a user to be selecting or enabling this?

You might want to consider increasing the test coverage by temporarily touching west.yml.

Reply:
Not needed I think. Openthread is using mbedtls entropy, not PSA entropy.